index.d.ts

```diff
@@ -31,6 +31,7 @@ export interface PointOfViewOptions {
   layout?: string;
   root?: string;
   viewExt?: string;
+  propertyName?: string;
```

Comment:
could you please add a test for this in https://github.com/fastify/point-of-view/blob/master/test/index.test-d.ts?